src/shared/globalPostStyles.js

```diff
@@ -27,22 +27,14 @@ export const GlobalPostStyles = createGlobalStyle`
     font-size: ${(props) => props.theme.post.fontSizes.base};
   }
 
-  p {
-    margin: 0 0 1em;
-  }
-
-  ul, ol {
-    margin: 1.25em 0;
-  }
-
-  ul > li,
-  ol > li {
-    margin: 0 0 .666666em
```

Comment:
Please add these back. This file is for blog post global styles. It is only being loaded on blog posts, need it for blog post template.